### ISSUE TITLE
[2.2.0] Wingpanel crashes upon attempting to summon calendar widget

### ISSUE THREAD

**Issue by dannios** [elementary/wingpanel-indicator-datetime#199]
## Prerequisites
- [x] I have searched open and closed issues for duplicates.

## Describe the bug
Clicking on datetime indicator to summon calendar widget will always crash wingpanel as seen here:

![Screen record from 2019-11-03 07 41 41](https://user-images.githubusercontent.com/35110421/68085735-1f545e00-fe12-11e9-9bc1-642f2e501e61.gif)

 
### To Reproduce
1. Click on date/time to bring up calendar widget.
2. Wingpanel will crash.

### Expected behavior
For the wingpanel calendar widget to appear.

### Logs
[Can be found here.](https://pastebin.com/p4yPTzhL)

For brevity's sake, this seems to be the issue:
`Nov 03 07:55:49 danny kernel: wingpanel[8788]: segfault at 20 ip 00007f02a67ae6d4 sp 00007ffcbcf8a3a8 error 4 in libdatetime.so[7f02a6799000+21000]`

## Platform Information

OS:
elementary OS Juno 
Running Linux 5.0.0-32-generic and GTK Version 3.22.30

Hardware: 
Lenovo Ideapad 320
Dual-Core AMD A12-9720P RADEON R7, 12 COMPUTE CORES 4C+8G
Advanced Micro Devices, Inc. [AMD/ATI] Wani [Radeon R5/R6/R7 Graphics] (rev c8)

Please check what applies:
-->
- [x] I'm using the latest released stable version

**Comment by ryonakano** [comment]
Do you connect any accounts like Outlook or Google in the Calendar app? It looks like the reporter of the same issue posted yesterday points out that they [set up Outlook account in Calendar](https://github.com/elementary/wingpanel-indicator-datetime/issues/198#issue-516519123). I don't have any accounts set up in Calendar and can't reproduce this issue, but if you set up one it might be related to how it manages them.

**Comment by dannios** [comment]
Yes, I do! I tried deleting then resubmitting the account details, and it fixed the problem temporarily but again continues to crash.

**Comment by paulo8448** [comment]
Actually the problem also happens the problem when we export a '.ics' file in my case I put one with the holidays from Brazil. One more tip I got around the problem I export that same file in google calendar so the calendar app synced and appears normal in the panel.

![Captura de tela de 2019-11-04 13-00-32](https://user-images.githubusercontent.com/33562708/68136283-c8c74c80-ff03-11e9-98a8-5120a449a3cd.png)

**Comment by shawn1thompson** [comment]
I'm having the same issue, but I only have a CalDAV account connected. Let me know if I can provide any details.
